rules/order.js

```diff
@@ -30,6 +55,7 @@ module.exports = {
         'box-sizing',
         'display',
         'overflow',
+        'vertical-align',
```

Comment:
This one, and even more so `overflow` seem misplaced to me. Two reasons for this:

1. Like my comment about `position`, it seems odd to put anything between `display` and display-based properties, like the `flex-` stuff.

2. Anecdotally, I generally treat `vertical-align` and especially `overflow` as an after-thought, like I write up a nice little layout that works for the common case, and add `overflow` to fix an edge case or `vertical-align` to get the spacing just right. Because of that, I'd rather have these pinned to the bottom than so close to the top.

Not a strong stance here, I'm curious to hear others' thoughts :)